src/orion/serving/storage_resource.py

```diff
@@ -0,0 +1,232 @@
+"""
+Module responsible for storage import/export REST endpoints
+===========================================================
+
+Serves all the requests made to storage import/export REST endpoints.
+
+"""
+import json
+import logging
+import multiprocessing
+import os
+import traceback
+import uuid
+from datetime import datetime
+from queue import Empty
+
+import falcon
+from falcon import Request, Response
+
+from orion.core.io.database import DatabaseError
+from orion.core.worker.storage_backup import dump_database, load_database
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
This should be perhaps configurable through the webapi config. Actually, I think using `orion -vv serve [OPTIONS]` would correctly set this logger to DEBUG.

Reply:
Indeed. Plus, I just realized that `logger` is not used in this module, so I removed it.